ros2/test/diagnostics/tests.py

```diff
@@ -11,18 +11,18 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-import os
-import signal
 import unittest
+from threading import Event
+from threading import Thread
 
 import launch.actions
 import launch_ros.actions
 import launch_testing.actions
 import launch_testing.asserts
 import launch_testing.markers
-import yaml
-
-bag_dir = os.path.join(os.environ['TEST_TMPDIR'], 'bag')
+import rclpy
+from diagnostic_msgs.msg import DiagnosticArray
+from rclpy import node
```

Comment:
IMO, `node` is too generic so `import rclpy.node` might be better.

Reply:
Went with a single `import rclpy` now.